docs/guides/run-elasticsearch-local.rst

```diff
@@ -0,0 +1,99 @@
+==========================================
+Enabling Elasticsearch on the local server
+==========================================
+
+Read the Docs has been using Elasticsearch which is a platform for distributed search and analysis of data in real time. To enable the search feature on your local installation, you need to install the elasticsearch locally and run the Elastic server. 
+
+Installation has been mainly divided into following steps.
+
+1. Installing Java
+------------------
+
+Elasticsearch requires JAVA 8 or later. Use `Oracle official documentation <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_. 
```

Comment:
Style nit: I don't know which one is correct (Java vs. JAVA) but please use the same style in the documentation.